src/js/extension.js

```diff
@@ -66,8 +66,124 @@ var Extension;
             // has a .base value pointing to the editor
             // owning us. has been given a .name if no
             // name present
-        }
-        // core API definition for an "extension" follows:
+        },
+
+        /* parent: [boolean]
+         *
+         * setting this to true will set the .base property
+         * of the extension to be a reference to the
+         * medium-editor instance that is using the extension
+         */
+        parent: false,
```

Comment:
i would almost default this to true for all the things. especially if most of the internal things already make the assumption the .base is there